frontend/lib/fhirUtils.ts

```diff
@@ -77,8 +77,40 @@ export const getCarePlan = (patient: Patient, conditions: Condition[], carePlanN
     }
 }
 
-export const getTask = (serviceRequest: ServiceRequest, primaryCondition: Condition, carePlanRef?: Reference): Task => {
+export const getTaskBundle = (serviceRequest: ServiceRequest, primaryCondition: Condition, patient: Patient, carePlanRef?: Reference): Bundle  & { type: "transaction" } => {
+    return {
+        resourceType: "Bundle",
+        type: "transaction",
+        entry: [
+            {
+                fullUrl: "urn:uuid:patient",
+                resource: {...patient, id: undefined},
+                request: {
+                    method: "PUT",
+                    url: `Patient?identfier=http://fhir.nl/fhir/NamingSystem/bsn|${getBsn(patient)}`
+                }
```

Comment:
I think it's probably safer to only create the `Patient` resource if it's new to the CPS. If it already knows the `Patient`, do not update it. Can also solve potential ETag/422 errors in certain FHIR servers. Something like this:

```suggestion
                request: {
                    method: "POST",
                    url: "Patient",
                    ifNoneExist: `identifier=http://fhir.nl/fhir/NamingSystem/bsn|${getBsn(patient)}`,
                }
```